src/libraries/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/DSA.cs

```diff
@@ -1125,5 +1125,168 @@ public int GetMaxSignatureSize(DSASignatureFormat signatureFormat)
                     throw new ArgumentOutOfRangeException(nameof(signatureFormat));
             }
         }
+
+        /// <summary>
+        /// Imports an RFC 7468 PEM-encoded key, replacing the keys for this object.
+        /// </summary>
+        /// <param name="input">The PEM text of the key to import.</param>
+        /// <exception cref="ArgumentException">
+        /// <paramref name="input"/> does not contain a PEM-encoded key with a recognized label.
+        ///   <para>
+        ///       -or-
+        ///   </para>
+        /// <paramref name="input"/> contains multiple PEM-encoded keys with a recognized label.
+        ///   <para>
+        ///       -or-
+        ///   </para>
+        /// <paramref name="input"/> contains an encrypted PEM-encoded key.
+        /// </exception>
+        /// <remarks>
+        ///   <para>
+        ///   Unsupported or malformed PEM-encoded objects will be ignored. If multiple supported PEM labels
+        ///   are found, an exception is raised to prevent importing a key when
```

Comment:
```suggestion
        ///   are found, an exception is thrown to prevent importing a key when
```

Reply:
It seems it is going to take a while longer for Ruby part of my career path to wear off.